### PR TITLE
Add ifstat and ntpupdate

### PR DESCRIPTION
**Pull request by LoyVanBeek**
Needed by https://github.com/ethz-asl/ros-system-monitor

### CODE REVIEW

**Comment by Alessandro-Barbieri** [comment]
`net-analyzer/ifstat` and `net-misc/ntp` in gentoo if you want to be more accurate

File: rosdep/base.yaml [hunk 1]
```diff
@@ -3398,9 +3398,11 @@ npm:
   fedora: [npm]
   ubuntu: [npm]
 ntp:
+  gentoo: [net-misc/ntp]
```
Comment (mikaelarguedas) [inline rosdep/base.yaml]: Is is possible to add [debian](https://packages.debian.org/search?keywords=ntp&searchon=names&suite=all&section=all) and [fedora](https://admin.fedoraproject.org/pkgdb/package/rpms/ntp/) here as well ?

**Comment by mikaelarguedas** [comment]
looks like travis is stuck, closing and reopening to rerun travis

**Review by mikaelarguedas — APPROVED** [review]
lgtm

**Pull request merged**